includes/admin/reporting/tables/llms.table.quiz.attempts.php

```diff
@@ -117,8 +119,12 @@ protected function get_data( $key, $attempt ) {
 				break;
 
 			case 'grade':
-				$value  = $attempt->get( $key ) ? $attempt->get( $key ) . '%' : '0%';
-				$value .= ' (' . $attempt->l10n( 'status' ) . ')';
+				$value      = $attempt->get( $key ) ? $attempt->get( $key ) . '%' : '0%';
```

Comment:
Again we probably don't want to show `0%` here if not actually finished the attempt.